guide/python/python_intro.md

```diff
@@ -0,0 +1,303 @@
+   example, when we set the variable `x = 5.0`, Python assigns systematically a float value 
+   (`5.0`) to a float variable (`x`), avoiding wastage of memory. 
+6. Object-oriented approach: Python supports an object-oriented environment (properties and behaviors are bounded into individuals objects), providing multiple functions and tools to developed applications with an object-oriented approach in mind. We will discuss this point in greater depth later.
+
+:::{admonition} Resources
+:class: seealso 
+You will be able to find more about Python's core strengths at: 
+* {ref}`Python webpage<python_web>`. 
+* {ref}`Python Tutorial<python_tutorial>`.
+* {ref}`Features of Python<python_features>`.
+* Discussion on why {ref}`Python is so popular<python_medium>`.
+:::
+
+## Data-structures in Python
+
+Python uses ___Data Structures___ as the method to handle and organize the data. They are the 
+fundamental building blocks around which one develops a Python program. Data Structures allow 
+the coder to store data, perform different operations and establish relationships among them. 
+Python provides multiple types of data structures with unique features to organize and 
+store the information efficiently according to the developer's needs. 
+
+:::{admonition} External libraries
+:class: tip
+_Raw_ Python offers different data structures that can easily be enhanced by using standard and 
+external libraries. In this chapter, we will focus only on the fundamental data types built into 
+core Python. We will give a small glimpse of enhanced data structures at the end of this section.
+:::
+
+Data Structures are classified into two main categories: _Primitive_ structures, which represent 
+the simplest blocks of data storage; and _Non-Primitive_ structures, which consist of more 
+advanced elements used in more specific and complex scenarios. 
+
+:::{figure-md} structure
+:width: 700px
+:align: center
+
+![structure](images/structure.png)
+
+__Python Data Structures__. These can be classified into two main groups: _Primitive_ Data 
+Structures, core building elements in Python, and _Non-primitive_ Data Structures, collections 
+of Primitive Data Structures. 
+:::
+
+### Primitive Data Structures
+
+Primitive Data Structures represent the most elementary building elements for storing pure and 
+plain data values in Python. They are defined in conjunction with a predefined set 
+of operations, allowing the manipulation of the stored data. Python has four types of Primitive Data Structures: 
+
+1. __Integer Data Structures__ are used to represent the integer numeric data. The maximum integer 
+   number one is able to reach is constrained by the memory of the system.  
+   
+1. __Float Data Structures__ are used for designating rational numbers. Float numbers in Python 
+   are represented as 64-bit (double-precision) values. The maximum value a float value can 
+   have is {math}`1.8 \times 10^{308}{math}`, while the minimum number one can represent is 
```

Comment:
```suggestion
   have is {math}`1.8 \times 10^{308}`, while the minimum number one can represent is 
```
* https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html?highlight=formulas#roles-an-in-line-extension-point
* https://myst-parser.readthedocs.io/en/latest/syntax/optional.html?highlight=math#math-shortcuts
*